whereby/sdk
# browser-sdk: Add support for pop out local video from grid

### Description

Adds support for moving the local video (pop out) of the grid. Functionality is similar to the web app. For now it's supported as a part of the participant menu, along with maximize and spotlighting, since these are "grid only" actions.

**Summary:**



**Related Issue:**



### Testing
1. Start storybook, and join the same room in a different tab
2. Go to the Video Grid story
3. Open the participant menu, and click the pop out button (Verify that this is only a button the **local** participant)
4. Verify that the local video cell is moved out of the grid, and down to the bottom right corner of the grid brounds
5. Open the participant menu on the popped out cell, and move it back to the grid. 
6. Verify that the local video is moved back to the grid



### Screenshots/GIFs (if applicable)
![image](https://github.com/user-attachments/assets/bccb3ca9-8d4c-42a6-9c04-db1d67b31303)
![image](https://github.com/user-attachments/assets/f182ce8e-c115-4586-bf88-c3ae265ec2e0)
![image](https://github.com/user-attachments/assets/cfe5ccc6-819f-44a7-b625-19ccf11e7fc3)



### Checklist

        under the terms of the MIT license.

### Additional Information
